src/app/app.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-root',
-  template: '<router-outlet></router-outlet>',
+  template: `<router-outlet></router-outlet>`,
```

Comment:
Czy ta zmiana jest potrzebna?

Reply:
Oryginalnie chciałem pracować na root, ponieważ myślałem, że ten komponent będzie stałym elementem strony, ale jednak lepiej, żeby to było osobno.